src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private String Dtime;
+    private LocalDateTime localDtime;
+
+    /**
+     * Constructor for Deadline object
+     * @param description for the activity
+     * @param s, time the activity is due
+     */
+    public Deadline(String description, String s) {
+        super(description);
+        this.Dtime = s; // accept "yyyy-mm-dd kkm" format
+        this.localDtime = LocalDateTime.parse(s.substring(1), DateTimeFormatter.ofPattern("uuuu-MM-dd kkmm"));
```

Comment:
Perhaps it might be better if the "Dtime" variable could be named more clearly as it may not be immediately obvious what the "D" stands for?